jwst/datamodels/schemas/combinedspectable.schema.yaml

```diff
@@ -11,7 +11,7 @@ datatype:
   datatype: float64
 - name: ERROR
   datatype: float64
-- name: SURF_BRIGHT
+- name: BACKGROUND
```

Comment:
That was the intent. Currently it is a bit confusing using surf_bright since depending on the type of data sometimes is it from the surf_bright column of extract 1d and sometimes it is from the background column. By changing it I was trying to make it completely clear what it was.  I hope this does not create too many new issues.

Reply:
We can NOT do this. The output of combine_1d must continue to contain `flux` and `surf_bright` columns. That's where all of the data of interest is stored when combine_1d is used to do things like combining the 1D spectra of WFSS sources.